projects/components/src/modal/modal.service.ts

```diff
@@ -10,18 +10,13 @@ import { ModalConstructionData, ModalContainerComponent } from './modal-containe
   providedIn: 'root'
 })
 export class ModalService {
-  private activeModal?: ModalRef<unknown>;
```

Comment:
Why? This was an intentional limitation so we don't start stacking modals on top of modals, do we want to do that now?

Reply:
+1 on the question.

Reply:
Sorry, I worked on this just before going on PTO. This PR is no longer needed. @itssharmasandeep  and others came up with a better solution, and the change is already merged. 

Closing this pr now.